frontend/src/utils/keep-layer-utils.ts

```diff
@@ -37,7 +38,7 @@ export function keepLayer(layer: LayerType, newLayer: LayerType) {
     return false;
   }
 
-  if (newLayer.type === layer.type && layer.type === 'anticipatory_action') {
+  if (newLayer.type === layer.type && isAnticipatoryActionLayer(layer.type)) {
```

Comment:
I believe this will only be needed for aa_drought

Reply:
Yes, this is only relevant for the aa_drought because it handles the fact that aa_drought can display the layer of each 'window' simultanously on the map. There is no concept of window in the aa_storm layer, as far as I know. I will fix that.